yarn.lock

```diff
@@ -0,0 +1,3485 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I think it will be good to continue using `npm` instead of `yarn` in this PR.

Reply:
https://github.com/StefanTerdell/zod-to-json-schema/pull/2/commits/bbd50d86699704e8679e4b6b52cadb868ac8871c fixed it